### PR TITLE
Bootstrap Liquidity RWA Lending Market

### PR DESCRIPTION
**Pull request by ImdioR**
-CP116
-Readme file

### CODE REVIEW

**Review by ImdioR — COMMENTED** [review]
- Update CP116
- Update Readme

File: cps/CP116/CP116.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Please keep in mind that this is a fixed rate supplied per day, so naturally, the overall reward decreases linearly as TVL increases. A fixed rate per day is the default and recommended structure from Morpho.
+
+### Change or improvement
+1. Initiate liquidity bootstrap and provide 60,000 CFG for 60 days once the market launches and deposit these rewards into the RWA Lending Market on the Morpho Rewards Distributor Contract [ 4eEqmbQMbFfNUg6bQnqi9zgUvQvSpNbUgstEM64Xq9FW58Xv ].
+2. Distribute 1000 CFG per day as rewards across the RWA Lending Market on Morpho.
+
+### Alignment to the mission of Centrifuge DAO
+By creating early liquidity incentives, we will bootstrap early participation in the market which will attract more lenders, borrowers and issuers. By capping the token rewards, we're limiting the Centrifuge out of pocket cost, while still supporting the initiatives that will help us build key infrastructure around Centrifuge. All of this serves the purpose of helping grow the TVL of Centrifuge, and maintaining our position as the largest RWA protocol.
+
```
Comment (Rhanok09) [inline cps/CP116/CP116.md:39]: Add following at the end of the file:

> Link to the RFC on the Forum: https://gov.centrifuge.io/t/cp116-rwa-lending-market-liquidity-bootstrap/6467
> 
> Link to the onchain vote (council motion + referendum):

File: README.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ This repository contains all submitted Centrifuge Proposals. The genesis proposa
 ## Proposed CPs
 |CP | Short title |
 |---|-------------|
+| [CP118](./cps/CP118.md)  | Runtime Upgrade 1103   |
+| [CP116](./cps/CP116/CP116.md)  | RWA Lending Market Liquidity Bootstrap   |
 | | |
```
Comment (Rhanok09) [inline README.md:72]: Delete the last three | | |

File: cps/CP116/CP116.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+cp: 116
+Title: RWA Lending Market Liquidity Bootstrap
+authors: @Imdior, @GraysonAlto
+uses-component: cp4, cp2
+technical-proposal: no
+requires-onchain: yes
+beneficiary: Contract Wallet Address
+wallet: 4eEqmbQMbFfNUg6bQnqi9zgUvQvSpNbUgstEM64Xq9FW58Xv
+impacts/modifies: -
+status: rfc
```
Comment (Rhanok09) [inline cps/CP116/CP116.md:11]: Status is going to be "voting" once this is merged.

**Review by Rhanok09 — APPROVED** [review]

**Pull request merged**